lib/cli/src/frameworks/lit/js/Header.stories.js

```diff
@@ -0,0 +1,16 @@
+import './Header';
+import { html } from 'lit';
```

Comment:
For consistency with the rest
```suggestion
import { html } from 'lit';

import './Header';
```